### DISCUSSION TITLE
Fragmented genomes biased

### DISCUSSION THREAD

**Discussion by francicco** [hahnlab/CAFE5#80] [General]
Hi,

I'm running CAFE on a dataset of 64 butterflies. Some of the genomes I'm using are more fragmented than others that are at the chromosome level. I noticed that the terminal branches for these fragmented genomes have always more loss than gain, and I think it's a clear bias. I think I need to at least compute λ without these. Is there a way in CAFE to do that? or do I have to run CAFE only on the good genomes, compute λ using those and attempt to apply it to the full dataset?
What do you advise in cases like this?

Thanks a lot
Francesco

**Comment by benfulton** [comment]
The biologists think you should leave those out, or use an error model specific to those species. It should be straightforward to set up an error model biased toward loss on those tips. But I'm not exactly sure what the last part means. You want to calculate lambda and then do what to the messy genomes? What does "apply it to the full dataset" mean?

**Comment by francicco** [comment]
Hi @benfulton, I considered leaving them out, but I'd remove a lot of useful information, I was more interested in the error model you're talking about. How can I implement it? Can you help me with that?

Maybe I didn't explain myself clear. I was thinking to estimate λ without the fragmented genomes, and then using that estimation for the full dataset.

Thanks a lot
F

**Comment by benfulton** [comment]
An error model consists of modifications of probabilities of moving from one family size to another through the tree. Provide a file with the probabilities. The file is structured as a series of lines containing the family size, the probability of moving to less than that size, the probability of that size staying the same, and the probabilities of the size becoming larger. Two header lines must be included: the maximum family size to process, and the differential of the probabilities.

        maxcnt:90
        cntdiff -1 0 1
        0 0.00 0.95 0.05
        1 0.05 0.9 0.05
        2 0.05 0.9 0.05
        3 0.05 0.9 0.05
        4 0.05 0.9 0.05

**Comment by francicco** [comment]
Thanks for that. Can I have an example how to implement it? I'm not sure I understand it.
Best,
F
